### DISCUSSION TITLE
Cognition.run

### DISCUSSION THREAD

**Discussion by dimitrats94** [jspsych/jsPsych#1165] [Q&A] [answered]
Hello everyone!

I want to use cognition.run for a pilot experiment we plan to run soon. I have written the code with JsPsych (it's a pretty simple reaction time experiment with welcome, 4 sound trials and a performance summary at the end) and it seems to work fine. 

The problem is that when I try to run it with cognition it fails to load. I have uploaded all stimuli (pictures and audio) and I cannot understand what I'm doing wrong. If I run each var individually it works fine, but when I add var timeline = []; at the beginning and timeline.push for each var it "gets confused". I'm using the Simple Reaction Time Task that is uploaded there as a prototype and my code doesn't seem to differ from it. 

Do you have an idea what I'm doing wrong? 

Thanks in advance! :)

**Comment by jodeleeuw** [comment]
Hi @dimitrats94,

Can you share the link to your experiment?

**Reply by dimitrats94** [reply]
Yes of course! @jodeleeuw 

Here it is:
https://ak8frjpgze.cognition.run

**Comment by becky-gilbert** [comment, marked as answer]
~~One issue is that you need to change the stimuli file paths so that the path just contains the file name, because your directory structure isn't maintained when you upload stimuli files to cognition.run.~~ ~~In your study, it looks like the audio files paths are e.g. "jsPsych/audio/Item04d.wav" etc., but on cognition.run they should just be the name of the file: "Item04d.wav" etc.~~

Edit: I was wrong, this wasn't the problem. Sorry!

**Reply by dimitrats94** [reply]
Hello @becky-gilbert, 

Thanks for the tip! I think I just corrected everything, but the experiment still fails to load. Fyi the stimuli loaded fine, when I added the variables one by one, there was no problems with that.

**Reply by jodeleeuw** [reply]
It looks like it is failing to load the file `Item01a.mp3`, which is the stimulus for one of your trials. The other audio files have a `.wav` extension. Maybe this is a typo?

**Reply by dimitrats94** [reply]
@jodeleeuw You're right, it was a typo. I just fixed it and it loads, but only the two first pages.

**Reply by jodeleeuw** [reply]
Since you are using the instructions plugin on the second page, the default key to press to continue is the right arrow. If you press that key the experiment continues. You might want to switch this to the html-keyboard-response plugin since you only have a single page of instructions.

**Reply by dimitrats94** [reply]
I see, thank you very much for the help!
